extensions/standard-processors/tests/unit/GetFileTests.cpp

```diff
@@ -40,9 +40,9 @@ namespace {
 class GetFileTestController {
  public:
   GetFileTestController();
-  [[nodiscard]] std::string getFullPath(const std::string filename) const;
+  [[nodiscard]] std::string getFullPath(const std::string& filename) const;
   [[nodiscard]] std::string getInputFilePath() const;
-  void setProperty(const core::Property& property, const std::string& value);
+  void setProperty(const core::Property& property, const std::string& value) const;
```

Comment:
I agree, it should not mark in this case it according to the documentation: _this check will not suggest to add a const to a non-const method if the method reads a private member variable of pointer type because that allows to modify the pointee which might not preserve logical constness_ To avoid this I made the members private in 942ae453a3b45e8ac7eed232be0283e415a890de